src/test/java/SYSC4806/AllTests.java

```diff
@@ -0,0 +1,21 @@
+package SYSC4806;
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+import org.junit.runners.Suite.SuiteClasses;
+
+import SYSC4806.Model.CategoryTest;
+import SYSC4806.Model.CourseTest;
+import SYSC4806.Model.LearningOutcomeTest;
+import SYSC4806.Model.ProgramTest;
+
+@RunWith(Suite.class)
+@SuiteClasses({ CategoryTest.class,
+        CourseTest.class,
+        LearningOutcomeTest.class,
+        ProgramTest.class,
+        ApplicationTest.class,
+        RepositoryTest.class })
+public class AllTests {
+
```

Comment:
Just re-running all the tests again?

Reply:
I agree, the branch should be deleted as Travis CI will run all the tests when you push automatically. You can also run all the tests locally by running Maven Tests like this: 
![image](https://user-images.githubusercontent.com/15951317/53738949-98a04900-3e5e-11e9-8bc6-962cb24acb59.png)

When creating a feature branch, this is generally the flow. 
1. Checkout master and pull any changes from the remote. 
2. Checkout a new branch for the feature you are developing.
3. While developing on your feature branch, run "git rebase master" to ensure your feature branch is up to date with master (in case of pull requests that go in while you are still working). All rebase does is pull's the changes from master to your branch and then replay's your branches commits onto it again. You may have to resolve some conflicts after this step. 

General rule of thumb is to always work your branch from master, you should not be working on branches started by other people or start branches from other people's branches as that will just make it more complicated for you. Hope this helps! :)

Reply:
THANKS FAMSS!